scripts/travis/build-all-in-one-image.sh

```diff
@@ -7,6 +7,9 @@ BRANCH=${BRANCH:?'missing BRANCH env var'}
 # be overrided by passing architecture value to the script:
 # `GOARCH=<target arch> ./scripts/travis/build-all-in-one-image.sh`.
 GOARCH=${GOARCH:-$(go env GOARCH)}
+# local run with `TRAVIS_SECURE_ENV_VARS=true NAMESPACE=$(whoami) BRANCH=master ./scripts/travis/build-all-in-one-image.sh`
+NAMESPACE=${NAMESPACE:-jaegertracing}
+ARCHS="amd64 arm64 s390x"
```

Comment:
Does travis support building images for `s390x` architecture ? Doesn't that means it will require base image of this architecture for alpine too ?

Reply:
Yes. We could build base images by `buildx` in next step.

golang and alpine both support `s390x` too. so the base images of jaeger could support `s390x`

the magic is `tonistiigi/binfmt` who installs `qemu`